apps/explorer/src/const.ts

```diff
@@ -129,7 +129,7 @@ export const NATIVE_TOKEN_PER_NETWORK: Record<SupportedChainId, TokenErc20> = {
 }
 
 export const TENDERLY_API_URL = 'https://api.tenderly.co/api/v1/public-contract'
-export const DEFAULT_IPFS_READ_URI = process.env.REACT_APP_IPFS_READ_URI || 'https://cloudflare-ipfs.com/ipfs'
+export const DEFAULT_IPFS_READ_URI = process.env.REACT_APP_IPFS_READ_URI || 'https://ipfs.io/ipfs'
```

Comment:
We have our own IPFS, so we could use that one. But maybe is not for now I agree with your solution, cause I know we experiencing some issues, so we need to review why the gateway gets permission issues.